### PR TITLE
Bash script for creating py-utils test rpm.

### PR DESCRIPTION
**Pull request by veerendra-simha-garikipati**
Signed-off-by: Veerendra Garikipati <veerendra.garikipati@seagate.com>

# Problem Statement
- Bash script for creating py-utils test rpm.

# Design
-  https://jts.seagate.com/browse/EOS-27312

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: N
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

-----
[View rendered py-utils/test/README.md](https://github.com/veerendra-simha-garikipati/cortx-utils/blob/EOS-27312/py-utils/test/README.md)

### CODE REVIEW

**Review by vastradparayya — APPROVED** [review]

File: jenkins/build_test_rpm.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline jenkins/build_test_rpm.sh:3]: 2021 shd be 2022

**Review by sachinpunadikar — APPROVED** [review]
Minor comment, rest all looks good

File: jenkins/build_test_rpm.sh (hunk 1 above)
Comment (veerendra-simha-garikipati) [reply jenkins/build_test_rpm.sh]: Done

**Review by ujjwalpl — APPROVED** [review]

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Pull request merged**